### ISSUE TITLE
Question: Words list is empty

### ISSUE THREAD

**Issue by booooooo7** [TinkoffCreditSystems/tinkoff-speech-api-examples#15]
Добрый день!
Ожидал увидеть список слов с временем конца и начала ([]*WordInfo в SpeechRecognitionAlternative), но как оказалось список пуст. Правильно я понимаю что в текущей версии API нет такой возможности?

Использовал пример на  golang.

**Comment by standy66** [comment]
Да, всё верно, на данный момент это заглушка, которая не заполняется. Мы планируем такую возможность добавить, поэтому пока оставлю issue открытым.
